packages/commonwealth/client/scripts/views/pages/CommunityHome/CommunityHomePage.tsx

```diff
@@ -0,0 +1,59 @@
+import { findDenominationString } from 'helpers/findDenomination';
+import { useFlag } from 'hooks/useFlag';
+import React, { useRef, useState } from 'react';
+import { useManageCommunityStakeModalStore } from 'state/ui/modals';
+import CWPageLayout from 'views/components/component_kit/new_designs/CWPageLayout';
+import { CWText } from '../../components/component_kit/cw_text';
+import { CWModal } from '../../components/component_kit/new_designs/CWModal';
+import ManageCommunityStakeModal from '../../modals/ManageCommunityStakeModal/ManageCommunityStakeModal';
+import IdeaLaunchpad from '../Communities/IdeaLaunchpad';
+import './CommunityHomePage.scss';
+
+const CommunityHome = () => {
+  const containerRef = useRef();
+  const tokenizedCommunityEnabled = useFlag('tokenizedCommunity');
+
+  const {
+    setModeOfManageCommunityStakeModal,
+    modeOfManageCommunityStakeModal,
+  } = useManageCommunityStakeModalStore();
+
+  const [selectedCommunityId] = useState<string>();
```

Comment:
I dont thing this needs to be in state. Or it will be modified in the upcoming prs?